frontend/src/pages/wifiPage/advancedConfigurationDialog/AdvancedConfigDialog.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import Dialog from "../../../components/atoms/dialog/Dialog";
+import PrimaryButton from "../../../components/primaryButton/PrimaryButton";
+import Spinner from "../../../components/atoms/spinner/Spinner";
+
+import styles from "./AdvancedConfigDialog.module.css";
+
+export type Props = {
+  active: boolean;
+  url: string;
+  onClose: () => void;
+  error: boolean;
+};
+
+export enum ErrorMessage {
+  AdvancedConfigError = "There was a problem opening the advanced network configuration. Please, close this dialog and try again later.",
+}
+
+export default ({ active, url, onClose, error }: Props) => {
+  const errorMessage = error && ErrorMessage.AdvancedConfigError;
+
+  return (
+    <Dialog
+      testId="advanced-config-dialog"
+      active={active}
+      title="Advanced Wifi Configuration"
+      className={styles.dialog}
+    >
+      <div className={styles.content}>
+          { !errorMessage && url === "" &&
+              <div className={styles.spinner}>
+                <Spinner size={80} />
+              </div>
+          }
+          { !errorMessage && url !== "" &&
+              <div className={styles.frameContainer}>
+                <iframe
+                  src={url}
```

Comment:
we should use the name/ip that the client is using to connect, not assume they can use pi-top.local:

![image](https://user-images.githubusercontent.com/7795213/180984620-5d1840d0-bec9-4c18-999e-f29683f20d33.png)

So I think we need to parse and adjust the url hostname, like we did for further also. You should be able to access this info on the server (`request.host_url` i think), or you could just send the port and path and do it on the client